alchemy/src/stripe/card.ts

```diff
@@ -0,0 +1,154 @@
+import Stripe from "stripe";
+import type { Context } from "../context.ts";
+import { Resource } from "../resource.ts";
+
+export interface CardProps {
+  customer: string;
+  source?: string;
+  number?: string;
+  expMonth?: number;
+  expYear?: number;
+  cvc?: string;
+  name?: string;
+  addressCity?: string;
+  addressCountry?: string;
+  addressLine1?: string;
+  addressLine2?: string;
+  addressState?: string;
+  addressZip?: string;
+  currency?: string;
+  defaultForCurrency?: boolean;
+  metadata?: Record<string, string>;
+}
+
+export interface Card extends Resource<"stripe::Card">, CardProps {
+  id: string;
+  object: "card";
+  brand: string;
+  country?: string;
+  dynamicLast4?: string;
+  fingerprint?: string;
+  funding: string;
+  last4: string;
+  tokenizationMethod?: string;
+}
+
+export const Card = Resource(
```

Comment:
please make sure to document each resource with various examples, take a look around the repo for examples and then apply that

also make sure to add a .md file in the alchemy-web/providers/stripe folder that mirror these examples. make sure to follow the same patterns established by the repo